Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -387,7 +387,7 @@ This block implements a rule-based tuning method for a PI or a PID controller, w
 steps:
```

Comment:
`This block implements a PI or PID controller with the control gains tuned by a rule-based method. If the tuning is enabled (<code>triTun=true</code>), the controller conducts the following steps to tune its gains:`